spec/controllers/case_workers/admin/case_workers_controller_spec.rb

```diff
@@ -12,7 +12,7 @@
     before { get :index }
 
     it 'returns http success' do
-      expect(response).to have_http_status(:success)
+      expect(response).to have_http_status(200)
```

Comment:
**Not a bocker** As discussed, we sometimes use `be_successful` and sometimes `have_http_status(200)`. could use one or other for consistency

Reply:
Switched to use `be_successful` in as many places as I could find and replace!